ouroboros-consensus/src/Ouroboros/Consensus/Storage/LedgerDB/InMemory.hs

```diff
@@ -667,9 +664,9 @@ ledgerDbCurrentValues =
 -- | The ledger state at the tip of the chain
 ledgerDbCurrent :: GetTip (l EmptyMK) => LedgerDB l -> l EmptyMK
 ledgerDbCurrent =
-    either unCheckpoint unCheckpoint
+    either unDbChangelogState unDbChangelogState
   . AS.head
-  . dbChangelogVolatileCheckpoints
+  . changelogVolatileStates
   . ledgerDbChangelog
 
 -- | Information about the state of the ledger at the anchor
```

Comment:
I cannot comment on un-edited lines, but I would also change `ledgerDbAnchor` to use `unDbChangelogState` and avoid `dbChangelogVolatileCheckpoints`

Reply:
Nick pointed out that it's better to use `dbChangelogVolatileCheckpoints` so that we use LedgerDB API functions without reaching for the internals.